extensions/jython/tests/src/com/google/refine/jython/JythonEvaluableTest.java

```diff
@@ -16,6 +21,31 @@
  */
 public class JythonEvaluableTest {
 
+  // Reproduces the situation when result is a PyObject
+  // Version with a test case which only calls the existing evaluate method
+  @Test
+  public void unwrapPyObjectTest(){
+    Properties props = new Properties();
+    Project project = new Project();
+
+    Row row = new Row(2);
+    row.setCell(0, new Cell("one", null));
+    row.setCell(0, new Cell("1", null));
+
+    props.put("columnName", "number");
+    props.put("true", "true");
+    props.put("false", "false");
+    props.put("rowIndex", "0");
+    props.put("value", 1);
+    props.put("project", project);
+    props.put("call", "number");
+    props.put("PI", "3.141592654");
+    props.put("cells", new CellTuple(project, row));
+    String funcExpression = "class Foo(object):\n      pass\n\nreturn Foo()";
+    JythonEvaluable eval1 = new JythonEvaluable(funcExpression);
+    eval1.evaluate(props);
```

Comment:
Perhaps it would be worth doing an assertion on the result of the evaluation? So that we make sure we don't return just *anything*?
For instance, you could check the value of a field on the object:
```python
class Foo(object):
    bar = 1

return Foo()
```
and then check that you can access the `foo` attribute via the `HasAttr` interface (not sure about the name, only from memory).

Reply:
Yeah, I think that would be cool